project4/index.html

```diff
@@ -0,0 +1,139 @@
+                make a similar case for how sustainable practices need to be implemented. They assert that it’s not
+                enough for us to “reduce, reuse, and recycle.” They explain how products need to be designed from the
+                outset so that after their lives they will provide nourishment for something new. McDonough and
+                Braungart feel that when designers employ the intelligence of natural systems—the effectiveness of
+                nutrient cycling, the abundance of the sun’s energy—they can create products and systems that allow
+                nature and commerce to fruitfully co-exist. Source for
+                text: http://www.ethicsingraphicdesign.org/morality/sustainability/</p>
+        </div>
+        <section id="schedule">
+            <h1>Schedule</h1>
+            <ol>
+                <li>Introduction 9:00-10:00</li>
+                <li>Workshop 10:00am-12:00pm</li>
+                <p>This workshop will deal with issues of sustainablility and waste in everyday life. On a personal level,
+                we will learn what, why, and how we can re-use and recylcle.</P>
+                <li>Discussion with Dr. Speaker Andrea Siodmok 12:00-1:00pm</li>
+                <p>Dr. Siomok will discuss policy changes and methods for graphic design as it relates to politics.</p>
+                <li>Break 1:00-2:30</li>
+                <li>Discussion with Speaker Nelson Soken 2:00-3:00</li>
+                <p>Soken will describe how to work with companies as a sustainable graphic designer.</p>
+                <li>Discussion with Speaker Ernesto Quinteros 3:00-4:00</li>
+                <p>Quinteros will discuss his sustainable projects and the philosophy behind them.</p>
+                <li>Closing Questions/ Networking 4:00-5:00</li>
+            </ol>
+        </section>
+        <section id="speakers">
+            <h1> Speakers</h1>
+            <ul>
+                <li class="andrea">
+                    <figure>
+                        <img src="images/speakerone.png" alt="First speaker's profile picture">
+                    </figure>
+                    Dr. Andrea Siodmok
+                    Lead for UK Policy Lab and Open Innovation Teams
+                </li>
+                <li class="nelson">
+                    <figure> <img src="images/speaker2.jpg" alt="second speaker's profile picture">
+                    </figure>
+                    Nelson Soken
+                    CEO, N.H. Soken Consulting
+                </li>
+                <li class="ernesto">
+                    <figure> <img src="images/speaker3.png" alt="second speaker's profile picture">
+                    </figure>Ernesto Quinteros
+                    Chief Design Officer, Johnson & Johnson
+                </li>
+            </ul>
+        </section>
+        <section id="register">
+            <h1> Registration</h1>
```

Comment:
See previous note about `h1`s
```suggestion
            <h2> Registration</h2>
```